rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml

```diff
@@ -44,6 +44,13 @@
   register: neutron_agentlist_result
   delegate_to: "{{ groups['utility_all'][0] }}"
 
+- name: Gather orchestration service list
+  shell: "source ~/openrc; openstack orchestration service list --insecure"
```

Comment:
Legit point, but this commit is only adding checking Heat the same way nova, neutron, and cinder is already being checked. For clarity, it duplicates the way the checks for the other services are done rather than re-implementing the test for Heat in a slightly different way, or rewriting existing checks.